common/main/af.xml

```diff
@@ -2576,12 +2573,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<zone type="Australia/Broken_Hill">
 				<exemplarCity>Broken Hill</exemplarCity>
 			</zone>
-			<zone type="Australia/Currie">
-				<exemplarCity>Currie</exemplarCity>
-			</zone>
 			<zone type="Australia/Melbourne">
 				<exemplarCity>Melbourne</exemplarCity>
 			</zone>
+			<zone type="Australia/Currie">
+				<exemplarCity>Currie</exemplarCity>
+			</zone>
```

Comment:
is this expected?

I thought the previous fixes would have included any such change.

Reply:
We ran CLDRModify -fp when testing the effect of the time zone changes — now this is without -fp — maybe that’s why we’re seeing additional locales affected